project/SparkBuild.scala

```diff
@@ -765,16 +750,6 @@ object CopyDependencies {
 
 }
 
-object Java8TestSettings {
-  import BuildCommons._
-
-  lazy val settings = Seq(
-    javacJVMVersion := "1.8",
-    // Targeting Java 8 bytecode is only supported in Scala 2.11.4 and higher:
-    scalacJVMVersion := (if (System.getProperty("scala-2.10") == "true") "1.7" else "1.8")
```

Comment:
yeah, if it's working for you, that's good enough. I guess we'll find out pretty quickly if something needs to be adjusted in the jenkins jobs.